mysql-test/include/has_tokudb.inc

```diff
@@ -0,0 +1,18 @@
+#
```

Comment:
Now we have have_tokudb.inc and has_tokudb.inc and it's really hard to tell the difference from the filename which one is supposed to do what. have_tokudb_optional.inc ?